shell/key-bindings.fish

```diff
@@ -1,52 +1,56 @@
 # Key bindings
 # ------------
 function fzf_key_bindings
-  # Due to a bug of fish, we cannot use command substitution,
-  # so we use temporary file instead
-  if [ -z "$TMPDIR" ]
-    set -g TMPDIR /tmp
-  end
 
-  function __fzf_escape
-    while read item
-      echo -n (echo -n "$item" | sed -E 's/([ "$~'\''([{<>})])/\\\\\\1/g')' '
+  # Use last token $cwd_esc as root for the 'find' command
+  function fzf-file-widget -d "List files and folders"
+    set -l cwd (commandline -t)
+    # The commandline token might be escaped, we need to unescape it.
+    set cwd (eval "printf '%s' $cwd")
+    if [ ! -d "$cwd" ]
+      set cwd .
```

Comment:
Yes, I get different results. Note that I use find command from macOS. I don't have the issue if I use gnu find from coreutils package.